### ISSUE TITLE
Remove 0 option from deer choices

### ISSUE THREAD

**Issue by chrissnyder** [zooniverse/Wisconsin-Zoo#13]
- Deer sub-choices currently have 7 options each, 0 to 6+. As only "finding" at least one deer in one of the categories is required, having the 0 present is superfluous.

![](http://i.imgur.com/QGL47EU.png)

**Comment by DarrenMcRoy** [comment]
As long as you don't HAVE to select one option from each of the four, right?

**Comment by chrissnyder** [comment]
No, you only need to specify at least one deer in total. So select "1" in any of the categories is sufficient,

**Comment by DarrenMcRoy** [comment]
Awesome, that's how it should be.
